### PR TITLE
small changes

### PR DESCRIPTION
**Pull request by shivanshsoni**

### CODE REVIEW

File: etc/updateCopyRightYear.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-#Updates the year in our copyright statement. i.e. "* (C) Copyright 2003-2016," gets replaced by "* (C) Copyright 2003-[current_year],"
+#Updates the year in our copyright statement. i.e. "* (C) Copyright 2003-2018," gets replaced by "* (C) Copyright 2003-[current_year],"
```
Comment (jsichi) [inline etc/updateCopyRightYear.sh]: There's no need to update this since the example is showing an out-of-date year intentionally.

Comment (shivanshsoni) [reply etc/updateCopyRightYear.sh]: ok sir.

Comment (shivanshsoni) [reply etc/updateCopyRightYear.sh]: Sir I have sent you an email mentioning solution of gsoc-warmup. Please check that.

**Comment by jsichi** [comment]
Thanks!

**Pull request merged**